dbms/src/Flash/tests/gtest_compute_server.cpp

```diff
@@ -12,12 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <AggregateFunctions/AggregateFunctionFactory.h>
```

Comment:
todo: add ColumnNullable case

Reply:
Yes, `ColumnNullable` case has already been added, check https://github.com/pingcap/tiflash/pull/9167/files#diff-3138850d8b2566955274caf9dc38803c00fea2ae0c71c295e938f9ec5fd65e34R1775

Reply:
but I didn't test `spill` and agg func like `group_concat` in unit test, because `group_concat` not support by test framework. need to test it in endless(WIP)